### ISSUE TITLE
Linux: Fuse directories not unmounted properly if Autoortho terminated with CTRL+C

### ISSUE THREAD

**Issue by squashplayer** [kubilus1/autoortho#406]
**Describe the bug**
When I close Autoortho by using CTRL+C in the Terminal, it termantes, but it doesn't clean up the Fuse directories it creates in the Custom Scenery folder

**To Reproduce**
1. Start Autoortho
3. Click Run
4. When running, press CTRL+C
5. repeat this several times
6. Eventually, you will notice that Autoortho cannot mount the sceneries. There will be an error message and some traceback to several .py scripts

**Expected behavior**
Expected: Autoortho can be terminated and run several times per Linux session

**Screenshots**

**(please complete the following information):**
 - OS: Linux Mint
 - XPlane Version: Beta 12.07
 - AutoOrtho Version 0.6.0
 - Filesystem type ext4 (Linux)

**Additional context**
I made a workaournd using the following shell script:

#!/bin/sh

fusermount -u ~/Desktop/XP12/Custom\ Scenery/z_ao_afr
fusermount -u ~/Desktop/XP12/Custom\ Scenery/z_ao_asi/
fusermount -u ~/Desktop/XP12/Custom\ Scenery/z_ao_aus_pac/
fusermount -u ~/Desktop/XP12/Custom\ Scenery/z_ao_eur/
fusermount -u ~/Desktop/XP12/Custom\ Scenery/z_ao_na/
fusermount -u ~/Desktop/XP12/Custom\ Scenery/z_ao_sa/

rm -r ~/Desktop/XP12/Custom\ Scenery/z_ao_afr
rm -r ~/Desktop/XP12/Custom\ Scenery/z_ao_asi/
rm -r ~/Desktop/XP12/Custom\ Scenery/z_ao_aus_pac/
rm -r ~/Desktop/XP12/Custom\ Scenery/z_ao_eur/
rm -r ~/Desktop/XP12/Custom\ Scenery/z_ao_na/
rm -r ~/Desktop/XP12/Custom\ Scenery/z_ao_sa/

cd ~/AutoOrtho
./autoortho_lin_0.6.0-1.bin

**Comment by jonaseberle** [comment]
> When running, press CTRL+C
> repeat this several times

Do not repeatedly press CTRL+C since that would kill the teardown process. Wait until autoortho has finished. There are intentional delays per scenery pack + some for the stat thread. So just expect it to take 10s until it finished. After that you can start it again.

**Comment by mercster** [comment]
Yeah this isn't a bug, this is you sending multiple SIGINT signals to several processes that are trying to clean up after the first ending process.

**Comment by squashplayer** [comment]
This can be closed, I had sent several CTRL+C signals, thanks!
